### ISSUE TITLE
Visual Chirality

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#42]
# TL;DR
The left-right flipping used in data augmentation assumes that flipping does not change the data distribution. However, in fact, the data distribution is a little different (most people wear the clock on their left hand side, but when the data is flipped, it appears to be worn on the right hand side, and the distribution is different from the original data with the clock on the left hand side.) ), the DL model can determine whether the image was flipped or not. 
![image](https://user-images.githubusercontent.com/24524018/90023811-1ffa1900-dcef-11ea-96ce-066b1de1257e.png)

# Why it matters:
There is a hope that the this can become a new tool by extracting something whose distribution changes by flip them in this way.

# Paper URL
https://arxiv.org/abs/2006.09512

# Submission Dates(yyyy/mm/dd)
2020/06/16

# Authors and institutions
Zhiqiu Lin, Jin Sun, Abe Davis, Noah Snavely
- Cornell University
- Cornell Tech

# Methods

# Results

# Comments

CVPR2020
